knack/index.js

```diff
@@ -384,43 +384,29 @@ $(document).on("knack-view-render.view_2587", function(event, scene) {
 
   // Add React app as iframe
   $(
-    '<iframe src="https://atd-geo-knack-ui.netlify.com/" frameborder="0" scrolling="yes" id="mapIFrame" \
+    '<iframe src="https://localhost:9001" frameborder="0" scrolling="yes" id="mapIFrame" \
     style="width: 100%;height: 523px;"></iframe>'
   ).appendTo($view_2587);
 
-  $view_2587.find("form").hide();
-
   // set up Post Message connection with iframe and parent page
-  //create popup window
   var iframe = document.getElementById("mapIFrame").contentWindow;
 
   function sendMessageToApp(message) {
     const stringifiedMessage = JSON.stringify(message);
+    console.log("inside API", stringifiedMessage);
     iframe.postMessage(stringifiedMessage, "*");
   }
 
-  // create lat/lon request button
-  $('<button id="latLonButton">Get Lat/Lon from Map</button>').appendTo(
-    $view_2587
-  );
-
-  // send message to iframe on button click
-  $("#latLonButton").on("click", function(e) {
-    var coordMessage = { message: "KNACK_LAT_LON_REQUEST" };
-    const message = JSON.stringify(coordMessage);
-    console.log("knack:  sending message:  " + message);
-    iframe.postMessage(message, "*"); //send the message and target URI
-  });
-
   // listen for response
   window.addEventListener("message", function(event) {
     console.log("message received:  " + event.data, event);
-    var latLonResponse = event.data.split(", ");
-    $view_2587.find("form").show();
-    var $latLonFields = $("#kn-input-field_3194");
+    const data = event.data;
```

Comment:
I'm on the fence if we should use `const` in knack.js files. It would break the app in IE10<
https://caniuse.com/#feat=const

Reply:
Good to know. I'll use var in the custom Knack JS so that we are using consistent keywords.